cmd/kube-proxy/app/options.go

```diff
@@ -303,9 +305,56 @@ func (o *Options) processHostnameOverrideFlag() error {
 		o.config.HostnameOverride = strings.ToLower(hostName)
 	}
 
+	if len(o.nodeIPOverride) > 0 {
+		o.config.NodeIPOverride = o.nodeIPOverride
+	}
 	return nil
 }
 
+// processIncompatibleFlags processes flags which can't be directly mapped to internal config.
+func (o *Options) processIncompatibleFlags(fs *pflag.FlagSet) {
+	fs.Visit(func(f *pflag.Flag) {
```

Comment:
This is weird...

Is this so that you can figure out which `Options` fields actually got set vs which ones just have the default values? Is there no better way to do that?

Reply:
Yes. Will try to find a better approach.

Reply:
have used https://pkg.go.dev/github.com/spf13/pflag@v1.0.5#FlagSet.Changed now